Storage/RemoteTabs.swift

```diff
@@ -89,6 +68,15 @@ public struct RemoteTab: Equatable {
     public func withClientGUID(_ clientGUID: String?) -> RemoteTab {
         return RemoteTab(clientGUID: clientGUID, URL: URL, title: title, history: history, lastUsed: lastUsed, icon: icon)
     }
+
+    public func toRemoteTabRecord() -> RemoteTabRecord {
```

Comment:
```suggestion
    public func saveRemoteTabRecord() -> RemoteTabRecord {
```

This is for saving remote tab record right? 🤔

Reply:
No, `toRemoteTabRecord` is an adapter function that is converting the iOS `RemoteTab` struct to a a-s `RemoteTabRecord`. `RustRemoteTabs.setLocalTabs` is where the saving is done.